mona_openai/mona_openai.py

```diff
@@ -72,6 +73,19 @@ def _get_mona_single_message(
     )
 
 
+def _init_mona_class(client, context_class_name, openai_endpoint_name):
+    response = client.create_openai_context_class(
```

Comment:
Maybe wrap in try-except (so we catch the exception of bad communication)?

Reply:
No need - already done in Mona's client. There's an env var that decides whether Mona errors should be raised or not.